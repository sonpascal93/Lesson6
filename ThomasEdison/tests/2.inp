20
-.--.-....-...--.--.-..---..--..--.
9
YLUVKNXG
PP
YLUVKNXGPP
Y
LUVKNX
G
PP
YLUVKNXGP
P
...-.-..-..-....-.---.-.-...--.-.-.---.-....-..---.-.-.-...-....
10
VLXSAJ
AAVCTJLFWCKIA
H
VLXSAJA
AVCTJLFWCKIA
H
VLXS
AJAAVCTJLFW
CKIA
H
..-.....---.--...--.-..-....-----.--...-...-.-.....-
12
FHOPUCLUMQNUSCSA
FHOPUCLUMQN
US
CSA
F
HOPUCLUMQ
NUS
C
SA
F
H
OPUCLUMQNUSCSA
--...-.--....-.-..--.-..-.......-.--.-.--.-.-...
14
TBYHTLTKFHF
QAC
B
TBYHTLTKFH
F
QA
C
B
TBYHTLTK
FHFQ
ACB
TBYHTLTK
FHFQAC
B
-..-..-..--.-....-..----.--..-
10
N
LXN
DFWQ
X
NLXNDFWQX
NLX
NDFWQX
NLXNDFWQX
NL
XNDF
..-.-.--.-...-.-.-----.-...--.
9
IC
MR
FY
OL
P
ICM
RFY
OL
P
..-..---.---.--....-..-..----.----.----.-.----..
17
FJAMPVUUTQM
Y
Q
J
D
FJAMPVUUTQMYQ
J
D
FJAMPVUUTQ
MYQ
JD
FJAMP
VUUTQMYQJ
D
FJAMPVUUTQMYQJ
D
FJAMPVUUT
....-..--.-.-.--..--..--..-.-..-.
4
SRWRNM
U
X
X
--.-..-.--..
9
QI
K
D
QI
K
D
QI
K
D
..-..---.--..-..---
1
F
----....-.......-..---..----..--.-.---....-..-.--.-.-..--...
17
MZIDSFWDODQ
JHD
YRXB
MZIDSFWDO
DQJHD
YR
XB
MZIDSFWDODQJHDYRXB
MZ
ID
SFWDO
DQJHDYR
XB
MZIDSFWDOD
QJ
HDYRX
B
--..-.--.--...------..--.-..---.--.-.-.-.---.---...--.-
4
ZYW
IJO
UCJWRKJ
JV
-.-.--..
4
C
Z
C
Z
.....-
12
IV
IV
IV
I
V
I
V
IV
IV
I
V
I
..-.-..-..--..-.......--..------.-...-
4
UA
UIGLHPJ
MKV
UAUIGLHPJMK
-.----.....-.-.-.---.....-.
15
TJBIN
KJSER
T
J
BINKJS
E
R
TJ
B
INKJSE
R
TJBI
NK
J
SE
...-...--.-..-.--..-..----..--.---.-.-..-.
12
VSQUWFJDQG
KF
VSQUWFJDQGKF
VS
QUWF
JD
Q
G
KF
VSQU
W
FJ
--....-----....-.---...-.-.-..-...-.--.---.----.
9
ZUOBAWBCX
SKKTYG
ZUOB
AWBCXSKKTYG
ZUOBA
WB
CXSKK
TYG
ZUO
..--...-.....--.-----.....---.-
11
ITDRS
PT
ODVQ
ITD
RSPTOD
V
Q
ITDRS
PTO
DV
Q
--.-.--.----....-..--..--.---.--.-.----..-....--....--...--.
18
GYJBLZMJAKWZBP
SZ
P
GYJBLZMJAKWZBPS
Z
P
GYJBLZMJAKWZBPS
Z
P
GYJBLZMJAKW
ZBP
S
ZP
GYJBLZMJAKW
ZBPSZP
GYJBLZMJA
KWZBPSZ
P
