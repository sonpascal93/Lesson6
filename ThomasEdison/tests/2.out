8
9
9
20
7
16
44
0
27
0
15
0
4
17
1
4102
5
3
8
63
